js/web/lib/index.ts

```diff
@@ -13,8 +13,12 @@ if (!BUILD_DEFS.DISABLE_WEBGL) {
   const onnxjsBackend = require('./backend-onnxjs').onnxjsBackend;
   registerBackend('webgl', onnxjsBackend, -10);
 }
+
 if (!BUILD_DEFS.DISABLE_WASM) {
   const wasmBackend = require('./backend-wasm').wasmBackend;
+  if (!BUILD_DEFS.DISABLE_WEBGPU) {
+    registerBackend('webgpu', wasmBackend, 5);
+  }
```

Comment:
Yes. Because WebGPU backend is enabled via JSEP, which depends on WASM. However, ort-web ( the TypeScript part ) does not use the build.py/cmake to build; it uses webpack and it's separated build.